extensions/amp-story/0.1/amp-story.js

```diff
@@ -409,6 +409,14 @@ export class AmpStory extends AMP.BaseElement {
       this.onDesktopStateUpdate_(isDesktop);
     });
 
+    this.storeService_.subscribe(StateProperty.CAN_SHOW_SYSTEM_LAYER_BUTTONS,
```

Comment:
Could you please add a todo that we should eventually move the "share pill" to the system layer so we can remove this code?
You can assign it to me if you want